drivers/vdpa/virtio/virtio_vdpa.c

```diff
@@ -1630,6 +1642,7 @@ virtio_vdpa_dev_do_remove(struct rte_pci_device *pci_dev, struct virtio_vdpa_pri
 
 	if (priv->configured)
 		virtio_vdpa_dev_close(priv->vid);
+	virtio_vdpa_doorbell_relay_disable(priv);
```

Comment:
Do we need this? if configured, it will be done in close. if not configured, it seems relay will not be enabled. Is there any case that this function call is necessary?

Reply:
just prevent if config fail, still check doorbell relay setting

Reply:
priv->configured flag is set in the end of config, just after relay_enable. relay enable fail will not return in current code. so if config fail, it's failing before enable. right?